app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -849,6 +849,12 @@ public void handleTouchEvent(MotionEvent aEvent) {
                 }
                 // Return to discard first click after focus
                 return;
+            } else {
+                // In case the window has already been active from a previous event,
```

Comment:
If we remove the call to `setActiveWindow(true)` below, is this still needed?

See my note above for an alternative.